### PR TITLE
GitHub actions

### PR DESCRIPTION
**Pull request by Tristramg**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+      - name: Install stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+
+      - name: Run cargo test
+        uses: actions-rs/cargo@v1
```
Comment (antoine-de) [inline .github/workflows/tests.yml:40]: can we also test the non default features?

Comment (Tristramg) [reply .github/workflows/tests.yml:40]: maybeeeeee, I’ll dive into the documentation tomorrow and ping you again

Comment (Tristramg) [reply .github/workflows/tests.yml]: done, not it tests on all features

**Comment by Tristramg** [comment]
And I added a cache, might save a few seconds

**Review by antoine-de — APPROVED** [review]

**Pull request merged**